jvm/src/main/scala/io/glutenproject/GazelleJniConfig.scala

```diff
@@ -209,6 +209,9 @@ class GazelleJniConfig(conf: SQLConf) extends Logging {
   val shuffleSplitDefaultSize: Int =
     conf.getConfString("spark.oap.sql.columnar.shuffleSplitDefaultSize", "8192").toInt
 
+  val loadch: Boolean =
+    conf.getConfString(GazelleJniConfig.OAP_LIB_PATH).contains(GazelleJniConfig.LIBCH_NAME)
```

Comment:
A default value is needed here. Otherwise, below error will be thrown on other backends:
```
java.util.NoSuchElementException: spark.oap.sql.columnar.libpath
  at org.apache.spark.sql.internal.SQLConf.$anonfun$getConfString$3(SQLConf.scala:3720)
  at scala.Option.getOrElse(Option.scala:189)
  at org.apache.spark.sql.internal.SQLConf.getConfString(SQLConf.scala:3720)
  at io.glutenproject.GazelleJniConfig.<init>(GazelleJniConfig.scala:213)
```